model/src/commonMain/kotlin/User.kt

```diff
@@ -19,19 +19,16 @@
 
 package org.eclipse.apoapsis.ortserver.model
 
+import kotlinx.serialization.Serializable
+
 /**
  * A data class representing a user.
  */
+@Serializable
```

Comment:
Can you explain why the class now needs to be serializable and why the removed properties are not required anymore?

Reply:
I think the model class is now used properly, and the reason for omitting data was explained in my commit message.